### DISCUSSION TITLE
Wizard template RFC

### DISCUSSION THREAD

**Discussion by serifluous** [twilio-labs/paste#3570] [Review request]
# Request for comment (RFC): Wizard template

_tl;dr Please leave feedback by Friday, October 27 on:_

- _The [Wizard spec](https://docs.google.com/document/d/1tHZ3J9Vkq52HmA5UJ_NPgq9Yjl__IeBUDehjOaULkJs/edit?usp=sharing) in the Google doc_
- _The [Wizard design](https://www.figma.com/file/EzNjCIMgA3PaCGrFiKAUcG/%F0%9F%A7%99-Wizard-pattern?type=design&node-id=364-115486&mode=design) in the Figma file. You can currently use the Figma component through the [Paste Patterns & Templates](https://www.figma.com/file/S4z0Kqjb9AYosnkRQldLte/Paste-Patterns-%26-Templates?type=design&node-id=6577-335654&mode=design&t=NtKPUq5HfMeuaGi8-4) library_

## Spec
Paste is adding a Wizard template and would love input from all roles.

Check out the proposed guidelines here: https://docs.google.com/document/d/1tHZ3J9Vkq52HmA5UJ_NPgq9Yjl__IeBUDehjOaULkJs/edit?usp=sharing

Please let us know how well (or not) these guidelines work for your product.

## Design

Review the design proposal below by dropping comments in the Figma file: https://www.figma.com/file/EzNjCIMgA3PaCGrFiKAUcG/%F0%9F%A7%99-Wizard-pattern?type=design&node-id=411-21708&mode=design&t=0k5rAf7zTwtQIzrn-0

Specifically:

- Does the proposed design work for existing use cases in your product?
- Are there important layout considerations that don't work with the proposed design?
- When are Breadcrumbs helpful in a Wizard flow (or any experience that heavily uses form elements)?
- For the Figma component, is it more helpful to have the component be a single page of the Wizard, or as a set of multiple pages?

Figma components for Wizard can already be used through the [Paste Patterns & Templates](https://www.figma.com/file/S4z0Kqjb9AYosnkRQldLte/Paste-Patterns-%26-Templates?type=design&node-id=6577-335654&mode=design&t=NtKPUq5HfMeuaGi8-4) library so you can test whether the design works well for your product.

![image](https://github.com/twilio-labs/paste/assets/3821840/580610bf-42be-48ba-8ba8-cd2ec1e24e85)

Thanks!

cc @andoyle
